includes/ApplHandlingByStr.php

```diff
@@ -74,6 +74,10 @@ function showFullStatus( $stat_str )
 		{
 			echo "<td><b>FORWARDED</b> from ".$str[0]."</td>";
 		}
+		else if( $str[1] == 'G' )
+		{
+			echo "<td><b>GENERATED</b> by ".$str[0] ."</td>";
+		}
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Expected 1 space after closing brace; newline found

<!-- Comment by [SideCI](https://sideci.com) -->